content/en/account_management/guide/usage-attribution-migration.md

```diff
@@ -0,0 +1,457 @@
+version of the daily zip files available
+from [Get specified daily custom reports][3]
+.
+
+The [Get hourly usage attribution][8]
+API provides this same data.
+
+As the hourly data files are very similar to files available
+from [Get specified daily custom reports][3]
+, the same guide applies, except the recommendation for time ranges. To migrate from the v1 monthly files,
+request all pages for each day in the month. Requests are limited to 24 hours at a time in the v2 API.
+
+### Monthly summary by tag files
+
+The monthly summary files use the naming format `summary_<tag>_<date>.tsv`. They provide a rollup for all usage across
+the month for each tag. The [Get monthly usage attribution][7]
+API provides this same data.
+
+See the sections below for differences between the v1 API and v2 API and recommendations for migrating to the v2 API.
+
+#### Response format
+
+The v1 API response contains a link to a ZIP file, containing a TSV file for each chosen tag.
+
+The v2 API response returns the usage attribution data in JSON format.
+
+To migrate to the v2 API, your processes must handle the data in JSON format. You can apply transformations
+as needed to the JSON data to create the format that best suits your needs.
+
+#### Tag breakdown
+
+In the v1 API, there is a separate TSV file for each chosen tag.
+
+In the v2 API, you can select the tag breakdown by supplying a tag configuration in `tag_breakdown_keys` as a comma-separated list.
+
+To migrate to the v2 API, make requests with each tag specified individually in `tag_breakdown_keys`.
+
+#### Tag values
+
+In the v1 API, if a resource is tagged with the same tag multiple times, it appears as a pipe (`|`) separated string
+in the column of the tag.
+
+Example:
+
+```
+month   public_id       team        infra_host_usage ....
+2022-01 abc123          billing|sre 100
+...
+```
+
+In the v2 API, the value corresponding to each tag key in the `tags` object is an array. If a resource is tagged with the
+same tag multiple times, there are multiple items in this list.
+
+Example:
+
+```
+...
+  "tags": {
+    "team": [
+      "billing",
+      "sre"
+    ]
+  },
+...
+```
+
+To migrate to the v2 API, your processes must handle resources with the same tag applied multiple times.
+Tag values in the v2 response array appear in the same order as they appear in the pipe-separated string in the v1
+response, so you can join the array with pipe characters to produce the same tag values as the v1 response.
+
+#### Total usage
+
+In the v1 API, the second row of the file contains aggregated usage for all tags.
+
+In the v2 API, the `metadata.aggregates` section of the response contains aggregated usage for all tags.
+
+To migrate to the v2 API, retrieve total usage from the `metadata.aggregates` section.
+
+#### Usage data type
+
+In the v1 API, some usage is returned with decimal precision. Example:
+
+```
+... container_usage ...
+... 55.4            ...
+...
```

Comment:
```suggestion
container_usage
55.4
```

Reply:
Not sure what the `...` fields are intended to represent, so let me know if this is expected or not. I removed them in my local build and took a screenshot if you're curious how it looks without them:
https://a.cl.ly/ApuXdvq6

Reply:
confirmed we can remove the ellipsis to avoid confusion